### DISCUSSION TITLE
Can copilot support Xcode? Xcode also supports plug-ins

### DISCUSSION THREAD

**Discussion by cuteworld** [community/community#13859] [Copilot Conversations] [answered]
Xcode can write c, c++, oc, swift, we need copilot to support Xcode

**Comment by inttpa** [comment]
The advice given to you here is: use Appcode :)

**Reply by cuteworld** [reply]
sry,appcode depend xcode

**Reply by cmp-nct** [reply]
Appcode has a plugin, xcode not ..

**Reply by mash3l777** [reply]
I believe Xcode support Plug-in now

**Reply by marciogranzotto** [reply]
AppCode was sunsetted https://blog.jetbrains.com/appcode/2022/12/appcode-2022-3-release-and-end-of-sales-and-support/

**Comment by chalemen** [comment]
Todo lo ise por medio de un solo teléfono no use laptop ni table ni algo parecido mañana si gustan alas 14:00 horas me conectaré y veré si en verdad han ayudado a mi México querido mi país ya ucrania y alos que fueron afectados me da gusto que se parara laguerra!!Pero si algún país kiere o anuncia otra guerra no solo le quitaré su capital si no que dejaré limpia su tarjeta madre con millas de archivos para su país e infección a sus apps para que le sea difícil pararse y no me gaste nada de dinero de finanzas lo tengo todo en mi cuenta por si gustan mañana se los daré para apoyar alas personas con pocos recursos y lo siento por tantos problemas pero ya me canse de ver muertes a familias con un gran futuro por delante arruinado solo por querer mas poder eso no va y mucho gusto me llamo Carlos y soy de guadalajara jalisco y mañana doy mis calles y número de casa por si gusta pasar por mi para aveŕ si me contrata alguien para ayudar al futuro del mundo para ser mejor....yo tengo pocos recursos trabajo de guardia de seguridad orita jajajaja pero siempre e visto como crecer y espero ver al mundo crecer con migo y aki tienen todo mi apoyo aun que no sea mucho lo poco que tengo es tuyo y mio!¡!!

**Comment by brozrost** [comment]
Don't think it's possible yet

**Comment by devm33** [comment, marked as answer]
## 👀  [GitHub Copilot code completion in Xcode is now available in public preview](https://github.com/orgs/community/discussions/141898) 🚀
